### ISSUE TITLE
Running the backfill daemon results in error if there's no last run

### ISSUE THREAD

**Issue by rexledesma** [dagster-io/dagster#3905]
## Summary
https://sourcegraph.com/github.com/dagster-io/dagster/-/blob/python_modules/dagster/dagster/core/execution/backfill.py#L212-219

```
dagster.check.ParameterCheckError: Param "run_id" is not a str. Got None which is type <class 'NoneType'>.
  File "/usr/local/lib/python3.7/site-packages/dagster/daemon/daemon.py", line 94, in _run_iteration
    next(daemon_generator), tuple([SerializableErrorInfo, CompletedIteration])
  File "/usr/local/lib/python3.7/site-packages/dagster/daemon/daemon.py", line 227, in run_iteration
    yield from execute_backfill_iteration(instance, grpc_server_registry, self._logger)
  File "/usr/local/lib/python3.7/site-packages/dagster/daemon/backfill.py", line 79, in execute_backfill_iteration
    instance, repo_location, backfill_job, chunk
  File "/usr/local/lib/python3.7/site-packages/dagster/core/execution/backfill.py", line 158, in submit_backfill_runs
    partition_data,
  File "/usr/local/lib/python3.7/site-packages/dagster/core/execution/backfill.py", line 218, in create_backfill_run
    instance.all_logs(parent_run_id),
  File "/usr/local/lib/python3.7/site-packages/dagster/core/instance/__init__.py", line 1012, in all_logs
    return self._event_storage.get_logs_for_run(run_id)
  File "/usr/local/lib/python3.7/site-packages/dagster/core/storage/event_log/sql_event_log.py", line 176, in get_logs_for_run
    check.str_param(run_id, "run_id")
  File "/usr/local/lib/python3.7/site-packages/dagster/check/__init__.py", line 258, in str_param
    raise_with_traceback(_param_type_mismatch_exception(obj, str, param_name))
  File "/usr/local/lib/python3.7/site-packages/future/utils/__init__.py", line 446, in raise_with_traceback
    raise exc.with_traceback(traceback)
```

## Additional Info about Your Environment

---
#### Message from the maintainers:
Impacted by this bug? Give it a 👍. We factor engagement into prioritization.

**Comment by prha** [comment]
Should be resolved with 1a27e64cd

**Comment by prha** [comment]
Fix was pushed out with `0.11.1`
